Azure/azure-cli
# [Cognitive Services] Support identity management and BYOX for Cognitive Services.

**Description**  
Add identity management for Cognitive Services.
Identity support is necessary for using user owned storage or encryption.

**Testing Guide**  
`az cognitiveservices account identity show` show the identity of an account, empty object will be displayed if no identity has been assigned.
`az cognitiveservices account identity assign` assign an identity to an account.
`az cognitiveservices account identity remove` remove the identity from an account, should still pass if account has no identity.

**History Notes**  


[Cognitive Services] az cognitiveservices account identity: Add identity management for Cognitive Services.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
